tests/modules/nf-core/bedtools/shift/nextflow.config

```diff
@@ -0,0 +1,10 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: BEDTOOLS_SHIFT {
+        ext.args = '-s 5'
```

Comment:
is `-s` a mandatory argument?

Reply:
From bedtools shift: 
`Usage:   bedtools shift [OPTIONS] -i <bed/gff/vcf> -g <genome> [-s <int> or (-p and -m)]`

Reply:
Allright it's fine as it is now :)